.github/workflows/legacy.yml

```diff
@@ -0,0 +1,182 @@
+name: ci
```

Comment:
Can we change the name of this to legacy? And maybe also the names of the individual stages as well?

Reply:
whoops. missed that

Reply:
does it look ok now?